spec/07-implicits.md

```diff
@@ -49,10 +49,14 @@ An _implicit parameter list_
 implicit. A method or constructor can have only one implicit parameter
 list, and it must be the last parameter list given.
 
-A method with implicit parameters can be applied to arguments just
-like a normal method. In this case the `implicit` label has no
-effect. However, if such a method misses arguments for its implicit
-parameters, such arguments will be automatically provided.
+The `implicit` modifier must be included in the first group of modifiers
+in the parameter list, but order does not matter: `override implicit`
+is the same as `implicit override`.
+
+A method with implicit parameters can be applied explicitly to arguments just
+like a normal method. If arguments are not supplied for some implicit
+parameters, implicit search will attempt to produce values for them.
```

Comment:
Implicit search only triggers if the argument list is missing entirely. If some arguments are specified, defaults will be used for the remaining ones (or "error: not enough arguments").

Reply:
Nothing good happens after midnight.